riot.im/README

```diff
@@ -0,0 +1,8 @@
+This directory contains the config files and deployment scripts for the official
+riot.im distribution of Riot Web at https://riot.im.
+
+You probably do not want to build with this config unless you're building the
+official riot.im distribution, but these files may be useful if you want to
+inspect the configuration used there.
+
+Riot Desktop uses a separate config (see electron_app/riot.im/config.json).
```

Comment:
perhaps add here how the deploy.py script is supposed to be used (e.g. that one should run it from the server, not your own machine).

Reply:
Ah right, good point. Added clarification.